### ISSUE TITLE
Error when trying to create same class in two namespaces

### ISSUE THREAD

**Issue by bourette** [intersystems-community/vscode-objectscript#569]
Using isfs for two namespaces, in the first namespace I did a right click and chose new file, entered MyCustom.FHIR.Interactions.cls, filed the code, everything is fine.  In the second namespace I did the same, but this time instead of giving me the new document I encountered an error, Unable to create file. 'isfs://dev:user/MyCustom.FHIR.Interactions.cls' that already exists when overwrite flag is not set.  Ensuring the class didn't exist with the explorer in this namespace I restarted VSCode and was able to file the new class.  

The code appears to be checking a cached value rather than checking with the database.

**Comment by gjsjohnmurray** [comment]
@bourette do you also get this problem if your isfs roots use the old ?ns=XXX notation rather than the new :xxx one?

**Comment by bourette** [comment]
Yes, when attempting to create the same class in a different namespace it indicates a file or folder already exists at this location. Please choose a different name.  Appears while typing in the name.

**Comment by gjsjohnmurray** [comment]
@bourette are you still able to reproduce this with the latest VS Code (1.55.2) and ObjectScript extension (1.0.9)?

I think a VS Code bug recently got fixed in the last release or two, relating to focus switching in the Explorer tree when the context menu was being used.

If the bug still exists for you would you be willing to record a short video of animated GIF to show it?
